homeassistant/components/tuya/climate.py

```diff
@@ -446,6 +446,12 @@ def swing_mode(self) -> str:
 
         return SWING_OFF
 
+    @property                                                        
+    def extra_state_attributes(self):
+        """Return device specific state attributes."""
+        LOGGER.debug("Device status:", self.device.status)
+        return self.device.status
```

Comment:
Attributes are not for debugging. If extra attributes are needed, they can be added in a controlled way (as in specific). Just dumping in all data is not what attributes are for.

What data are you trying to expose?

../Frenck

Reply:
@frenck in this particular case I will find the `valve` attribute useful for heating automation, but there are many other useful parameters, such as `power_state` - I will know if the batteries are not discharged

in general - I don't see any justification for hiding data.

Reply:
@frenck :
I've added filtering although as @Santanachia mentioned I don't see why hide the data. 
Depending on the firmware and the settings in the Tuya IoT portal different sets of data can be shown:
- standard instruction set (temp_set, temp_current, mode). This is missing some really usefull data, like valve position, valve setting, etc. 
- DP instruction set (used eg by Tuya Smart Home application). This set includes much more usefull data, including attributes that I use currently:
  - `fault` - to alert on fault
  - `roomtemp_calibrat` - to automate calibration using another temperature sensor.
  - `valve_set` (one of `normal`, `ForceOpen`, `ForceClose`) - allows tracking the valve state
  - `valve` - valve open percent. Allows (in conjunction with `valve_set`) detailed tracing of the Thermostat efficiency
  - `power_state` - to alert on batteries